### ISSUE TITLE
Changing the delimiter of the tag

### ISSUE THREAD

**Issue by Tecvoc** [vimwiki/vimwiki#928]
I have been using a custom script along with pandoc for converting vimwiki in markdown format to html, along with emojis. However emojis have a syntax very similar to tags ( like :star : for :star:) and they create new tags.

Is there is a way we can set the delimiter in the vimwiki global options for tags.

Currently I have identified as to where the change are needed, in order to do so, however due to limited vimscript knowledge I am just posting the diff code that worked for me.

```vimscript
syntax/vimwiki_markdown.vim

-let s:markdown_syntax.tag_search = '\(^\|\s\)\zs:\([^:''[:space:]]\+:\)\+\ze\(\s\|$\)'
+let s:markdown_syntax.tag_search = '\(^\|\s\)\zs@\([^@''[:space:]]\+@\)\+\ze\(\s\|$\)'

autoload/vimwiki/tags.vim
-     for tag in split(tag_group, ':')
+     for tag in split(tag_group, '@')

```

The above changes the delimiter to '@' like @mytag1@.

**Comment by tinmarino** [comment]
Related:  #922

**Comment by tinmarino** [comment]
Fixed: 188ead55db340f

See the new doc: https://github.com/vimwiki/vimwiki/blob/188ead55db340fc10e9d5facf40f0ecb42b81985/doc/vimwiki.txt#L3602-L3631

In your case ( after rebase on dev ):

```vim
let g:vimwiki_tag_format = {'pre_mark': '@', 'post_mark': '@', 'sep': '@'}
```
